app/src/main/java/com/amaze/filemanager/database/ExplorerDatabase.kt

```diff
@@ -280,7 +280,21 @@ abstract class ExplorerDatabase : RoomDatabase() {
                 )
             }
         }
-        internal val MIGRATION_8_9: Migration = object : Migration(8, DATABASE_VERSION) {
+        internal val MIGRATION_8_9: Migration = object : Migration(8, 9) {
+            override fun migrate(database: SupportSQLiteDatabase) {
+                database.execSQL(
+                    "UPDATE " +
+                        TABLE_CLOUD_PERSIST +
+                        " SET " +
+                        COLUMN_CLOUD_SERVICE +
+                        " = " +
+                        COLUMN_CLOUD_SERVICE +
+                        "+1"
```

Comment:
Is this solution still valid or will this give index out of bounds exception?

Reply:
To be fixed separately at #3629.